### ISSUE TITLE
How to read/retrieve section break in Word using office.js

### ISSUE THREAD

**Issue by shermaro91** [OfficeDev/office-js#3657]
Hello Team,

I have a requirement to analyze section breaks in a Word document, specifically to determine whether they are "Next Page" or "Continuous." Based on this information, I need to break the document into sections accordingly.

Is there an existing method or feature available in Office.js or any plans to introduce such functionality for reading and working with section breaks programmatically in Word documents?

Note : I don't want to use oneDrive due to some company policy.

My Environment
Platform : PC desktop
Host: Word
Office version number: Microsoft® Word for Microsoft 365 MSO (Version 2307 Build 16.0.16731.20052) 64-bit
Operating System: Windows 10 pro

**Comment by JHJ-MS** [comment]
Yes. Word JS API has [Section](https://learn.microsoft.com/en-us/javascript/api/word/word.section?view=word-js-preview) and [SectionCollection](https://learn.microsoft.com/en-us/javascript/api/word/word.sectioncollection?view=word-js-preview) APIs to support handling sections. Also we support [insert a section break](https://learn.microsoft.com/en-us/javascript/api/word/word.sectioncollection?view=word-js-preview).

**Comment by shermaro91** [comment]
Hi @JHJ-MS ,
I have used section for insert a section break.
But I didn't find any method to read the section break from document.
Can you just guide here with more explanation or any sample code?
Thanks again for quick response.

**Comment by shermaro91** [comment]
Hi @ZYUN-MSFT ,
Any updates on this?
We have requirement where we want to read, update and delete section break programmatically.
Please guide here if you have any inputs then.

If you want detailed use cases of this functionality than I can share you in personal mail.
As there are many things we can build on this if we got success in this.

Thanks

**Comment by JHJ-MS** [comment]
Hi @shermaro91, 
Yes, as far as I know, Office JS API doesn't support handling the section break directly. Please raise the New Feature Ask at https://techcommunity.microsoft.com/t5/microsoft-365-developer-platform/idb-p/Microsoft365DeveloperPlatform .
Feature requests on it are considered when we go through our planning process.

By the way, see whether [it](https://stackoverflow.com/questions/61107539/is-there-a-way-to-delete-a-section-break-in-word-js) is helpful for deleting section breaks.
